code/shapes.py

```diff
@@ -14,10 +14,11 @@ def construct(self):
         ring2 = Annulus(inner_radius=0.6, outer_radius=1, color=BLUE)
         ring3=Annulus(inner_radius=.2, outer_radius=1, color=BLUE)
         ellipse=Ellipse(width=5, height=3, color=DARK_BLUE)
-
+        
+        n = np.pi / 4
```

Comment:
Simplify `pi / 180 * 360 / 8` to `pi / 4`. Change `pi` to `np.pi` to avoid compiler error.